os/net/bluetooth/bt_netdev.c

```diff
@@ -69,6 +69,13 @@
 #include "bt_conn.h"
 #include "bt_ioctl.h"
 
+#ifdef CONFIG_BLUETOOTH_NULL
+int bt_dev_register(FAR const struct bt_driver_s *btdev)
```

Comment:
Actually register bt dev is not related to CONFIG_BLUETOOTH_NULL.
Below is correct example.
```
#ifdef CONFIG_BLUETOOTH_XXX //regarding net dev
bt_driver *bt_lower;
#ifdef CONFIG_BLUETOOTH_NULL
   bt_driver_s *bt_lower = bt_null_blahblah();
#elif CONFIG_BLUETOOTH_ANOTHER_MODEL
   bt_driver_s *bt_ lwer = bt_another_model_init_blahblah();
#else
   LOG("No lower device available, do not register here\n");
    return -ENXIO; 
#endif
   return bt_driver_register(bt_lower);
```

Reply:
I agree with you and this will be designed at later stage when clarity emerges with respect to driver.

Reply:
currently removed CONFIG_BLUETOOTH_NULL